src/vcs/git.rs

```diff
@@ -757,7 +757,7 @@ impl<'a> Browser<'a> {
     /// let mut browser = Browser::new(&repo, Branch::local("master"))?;
     ///
     /// // Clamp the Browser to a particular commit
-    /// let commit = Oid::from_str("223aaf87d6ea62eef0014857640fd7c8dd0f80b5")?;
+    /// let commit = Oid::from_str("a0dd9122d33dff2a35f564d564db127152c88e02")?;
```

Comment:
Why did this need to be done? :eyes: The reason I clamped it was so that we don't need to constantly update :smile:

Reply:
Hm, you're right. I wonder why the tests failed when I tried running them for the first time after bumping `git-platinum`?

Reverted this in: 301f49fc522e81e1f771629f3156b7a892b6d2ef.

Reply:
The commit count still had to be adjusted tho'...

https://github.com/radicle-dev/radicle-surf/pull/156/files#diff-eb83d6e8d5d1ac1c789414a0e87e97d7243260cd933b4d52d908644609bdbb85R934

Reply:
Makes sense :+1: